### PR TITLE
Add license, document run script and improve README

### PR DESCRIPTION
**Pull request by neelabhg**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## The MIT License (MIT)
+
```
Comment (msaxena2) [inline LICENSE.md:2]: Why not the UIUC/NCSA open source license?

Comment (neelabhg) [inline LICENSE.md:2]: Because the MIT license is more popular, at least in the Node.js community, which makes it easy for people to adopt this project.

**Comment by msaxena2** [comment]
@neelabhg That's all the more reason to use the UIUC/NCSA license. Getting
more people to notice it would be great.

**Comment by neelabhg** [comment]
GitHub's [choosealicense.com](http://choosealicense.com/) recommends the MIT license for being simple and permissive, which is probably why it is so popular on GitHub. Maybe they should recommend the NCSA license as well, at least for academic projects?

In any case, the MIT license to me seems more suitable for this project, as mentioned in my last comment.

I'll probably use the NCSA license for [currency-arbitrage-graph](https://github.com/neelabhg/currency-arbitrage-graph), as it seems well suited for it.

**Pull request merged**